index.html

```diff
@@ -145,24 +145,24 @@ <h1 class="title">About</h1>
 
     <section class="contact">
       <div class="headline">
-      <h2 class="title">Get Started</h2>
-      <p class="myname-intro"> I'm always interested in hearing about new projects, 
-                               so if you'd like to chat please get
-                               in touch.</p>
-      <form action="https://formspree.io/f/xgebneoy" method="post" class="contact-form">
+        <h2 class="title">Get Started</h2>
+        <p class="myname-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+
+      <form action="https://formspree.io/f/xgebneoy" method="post">
         <div class="form-row">
-          <label for="name">Your name</label>
-          <input type="text" id="name" name="name" maxlength="30" placeholder="Write your name..." required>
+           <label for="name">Your name</label>
+           <input type="text" id="name" name="name" maxlength="30" placeholder="Write Your name..." required>
         </div>
         <div class="form-row">
           <label for="email">Your email</label>
-          <input type="email" id="email" name="email" placeholder="Write your email..." required>
-        </div>
-        <div class="form-row">
-          <label for="contact-descrip">Your message</label>
-          <textarea name="contact-descrip" id="contact-descrip" cols="30" rows="10" maxlength="500" placeholder="Enter your message here..." required></textarea>
-        </div>
-        <button type="button" class="button-box">Start collaboration</button>
+          <input type="email" id="email" name="email" placeholder="Write Your email..." required>
+       </div>
+       <div class="form-row">
+        <label for="contact-descrip">Your message</label>
+        <textarea id="contact-descrip" name="contact-descrip" maxlength="500" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+       </div>
+         <button class="button-box" type="submit">Start collaboration</button>
       </form>
```

Comment:
- [OPTIONAL] You used the wrong style declarations for the hover and active pseudo-state of the button. I would recommend you use that provided by Figma. To get the correct style declarations, kindly look at the screenshot below that represents the section of your Figma where the styles are declared.

![image](https://user-images.githubusercontent.com/77106962/229896498-3cdcd5cc-a3d0-4493-ae51-fc553c01c78c.png)